backend/app/services/chat/url_parser.py

```
@@ -0,0 +1,740 @@
+                title=title,
+                content=markdown_content,
+                truncated=truncated,
+                size=len(response.content),
+            )
+
+        except Exception as e:
+            logger.exception(f"Error parsing webpage {url}")
+            return ParsedUrlResult(
+                url=url,
+                type=UrlType.WEBPAGE,
+                error=f"Failed to parse webpage: {str(e)}",
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)